namerd/storage/consul/src/main/scala/io/buoyant/namerd/storage/consul/ConsulDtabStoreInitializer.scala

```diff
@@ -45,6 +50,7 @@ case class ConsulConfig(
 object ConsulConfig {
   val DefaultHost = "localhost"
   val DefaultPort = Port(8500)
+  val DefaultBackoff = Backoff.decorrelatedJittered(1.millis, 1.minute)
```

Comment:
Was the default backoff (for `KvApi`) not previously `exponentialJittered` rather than `decorrelatedJittered`? Is there a motivation behind this change?

Reply:
BackoffConfig only produces `decorrelatedJittered` and `constant`.  Never `exponentialJittered`.  This is more consistent with elsewhere in linkerd.